docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.9'
+
+services:
+  postgres:
+    image: postgres:16-alpine
```

Comment:
just curious, if this is a ruby gem, why do we need postgres?

Reply:
The `pg` gem is not in the gemspec.  It is in the spec/Gemfile for testing only.  README guides user to add `pg` gem if needed in their own setup. (typically in their main application Gemfile)